client/extensions/woocommerce/app/dashboard/store-move-notice-view.js

```diff
@@ -22,53 +21,24 @@ import { recordTracksEvent } from 'calypso/state/analytics/actions';
 import megaphoneImage from 'calypso/assets/images/woocommerce/megaphone.svg';
 
 class StoreMoveNoticeView extends Component {
-	trackTryWooCommerceClick = () => {
-		this.props.recordTracksEvent( 'calypso_store_try_woocommerce_click' );
-	};
-
-	trackLearnMoreAboutWooCommerceClick = () => {
-		this.props.recordTracksEvent( 'calypso_store_learn_more_about_woocommerce_click' );
-	};
-
 	render = () => {
-		const { site, isStoreRemoved } = this.props;
+		const { site } = this.props;
 
 		return (
-			<Card
-				className={ classNames( 'dashboard__store-move-notice', {
-					'store-removed': isStoreRemoved,
-				} ) }
-			>
+			<Card className={ classNames( 'dashboard__store-move-notice', 'store-removed' ) }>
 				<img src={ megaphoneImage } alt="" />
 				<h1>{ translate( 'Find all of your business features in WooCommerce' ) }</h1>
 				<p>
-					{ isStoreRemoved
-						? translate(
-								'We’ve rolled your favorite Store features into WooCommerce. In addition to Products and Orders, you have top-level access for managing your Analytics, Marketing, and Customers. {{link}}Learn more{{/link}} about what has changed.',
-								{
-									components: {
-										link: (
-											<a
-												onClick={ this.trackLearnMoreAboutWooCommerceClick }
-												href="https://wordpress.com/support/new-woocommerce-experience-on-wordpress-dot-com/"
-											/>
-										),
-									},
-								}
-						  )
-						: translate(
-								'We’re retiring Store on February 22. With WooCommerce, discover a more flexible store management experience — including top-level access to your Analytics, Marketing, and Customers. {{link}}Learn more{{/link}} about what to expect.',
-								{
-									components: {
-										link: (
-											<a
-												onClick={ this.trackLearnMoreAboutWooCommerceClick }
-												href="https://wordpress.com/support/new-woocommerce-experience-on-wordpress-dot-com/"
-											/>
-										),
-									},
-								}
-						  ) }
+					{ translate(
+						'We’ve rolled your favorite Store features into WooCommerce. In addition to Products and Orders, you have top-level access for managing your Analytics, Marketing, and Customers. {{link}}Learn more{{/link}} about what has changed.',
+						{
+							components: {
+								link: (
+									<a href="https://wordpress.com/support/new-woocommerce-experience-on-wordpress-dot-com/" />
```

Comment:
@moon0326 Do we need to remove the tracking to learn more clicks?

Reply:
There is no reason to remove the tracking.

Reply:
@ilyasfoo  @mattsherman  I think it got removed while I was rebasing the code. I will add it back. Thank you for catching it 👍

Reply:
Added it back in https://github.com/Automattic/wp-calypso/pull/49641/commits/3b1b5854c0bb1b967bdd8a83a00111e79172784c

Reply:
No problem!